### ISSUE TITLE
Different plots after post-processing same data

### ISSUE THREAD

**Issue by kostasvar** [numbbo/coco#1839]
It seems that post-processing same data could produce different plots, using a newer version of the `cocopp` module.

**Comment by nikohansen** [comment]
See also #1007, which is however probably not related.

**Comment by brockho** [comment]
The corresponding plots are here: http://cocoexprm.gforge.inria.fr/LBFGS_comparison/ppdata/lbfgs_lbfgs/pprldmany-single-functions/pprldmany_f001_20D.svg and here: http://cocoexprm.gforge.inria.fr/ppdata_bbob_bbob-largescale/All_algorithms/pprldmany-single-functions/pprldmany_f001_20D.svg
(lbfgs and LBFGS are supposed to be the same data sets here)

**Comment by kostasvar** [comment]
Some more details about this:

For `bbob` data I checked the specific commit that we used for PPSN ([link](https://github.com/numbbo/coco/tree/786e3135d860ec4ef932858ad6217f3f5a94ee29)), a previous commit from the `development` branch at that period ([link]( https://github.com/numbbo/coco/tree/aba1b5334a149c538e2df705eff9bf5bb107de7a)) and the current `development` branch. The postprocessed results were the same, there was no shift for 1, 2 and 3 solvers. 

For `bbob-largescale` data, I checked that specific commit and the current `development` branch and  this horizontal shift appeared in all cases of 1, 2 and 3 solvers. The current plots are the correct ones after checking some data files.
